### PR TITLE
README readability improvements

### LINKED ISSUE #182: README needs updating re. urcap
In the root README, this sentence needs updating. It must have been copy/pasted from a different repo:

> you need to install the externalcontrol-1.0.5.urcap which can be found inside the resources folder of this driver or 

(the resources folder does not exist)

### PR DESCRIPTION
**Pull request by AndyZe**
I got an opportunity to use a UR hardware robot for the first time in ROS2  :+1:   These are some README improvements that would have helped me figure it out faster, I think.

- Add a Network Settings section with a good default setup.
- The URCap program is not located in the Resources folder, so delete that bit.
- Describe how to modify Host IP of the external_control URCap.
- Prompt the user to launch the URCap after the appropriate launch files. It's very easy to forget.

Fixes #182

### CODE REVIEW

**Comment by urrsk** [comment]
@AndyZe I had some problems making a PR for this PR. So I have send you an email.

**Comment by AndyZe** [comment]
@urrsk Received and updated. Thanks

File: README.md [hunk 1]
```diff
@@ -82,17 +82,55 @@ To use MoveIt some additional packages should be added into workspace:
...
 
 **Note:** For installing this URCap a minimal PolyScope version of 3.7 or 5.1 (in case of e-Series) is necessary.
 
 2. For installing the necessary URCap and creating a program, please see the individual tutorial on how to [setup a CB3 robot](/ur_robot_driver/doc/install_urcap_cb3.md) or how to [setup an e-Series robot](/ur_robot_driver/doc/install_urcap_e_series.md)
 
 3. On the remote PC, launch the suitable launch file which starts the robot driver and controllers (see details in [Usage](#usage) section).
 
-4. In the Program tab of the teach pendant, navigate to the URCaps section on the left and add the external control to the robot program by clicking on it. The program can then be executed by pressing the play button. Make sure the robot is turned on. The robot power status will be displayed on the bottom left.
+4. From the Program Robot tab of the teach pendant, load `external_control.urp`. Click on the "Control by..." section of the program to check the Host IP of the external PC. If it needs to be modified, make the modification under the Installation tab (as prompted on screen). You do not need to modify the Custom Port.
```
Comment (urrsk) [inline README.md:131]: Please verify that the same external_control.urp works  on both CB3 and e-Series. It should but please verify be we get a lot of issues

**Review by urrsk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AndyZe) [reply README.md:131]: @MGBla can you confirm that the teach pendant script you run on an e-Series is called `external_control`? Bonus if you can confirm it's v1.0.5 ([from here](https://github.com/UniversalRobots/Universal_Robots_ExternalControl_URCap/releases))

I can confirm it works for CB3 robots

Comment (AndyZe) [reply README.md:131]: OK, I got confirmation from a PickNik employee that they use the same script on a UR5e.

**Review by livanov93 — APPROVED** [review]

**Pull request merged**